layer/nrlf/producer/fhir/r4/strict_model.py

```diff
@@ -358,11 +358,42 @@ class RequestHeaderCorrelationId(BaseModel):
     ]
 
 
+class CodeableConcept(BaseModel):
```

Comment:
Why have this changes been added to the strict model in producer and not in consumer?

Reply:
there is no strict model in the consumer folder

Reply:
but it is in the base model of consumer

Reply:
My bad, I thought they both had the strict_model. 😅